pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MetadataAndDictionaryAggregationPlanClusterIntegrationTest.java

```diff
@@ -469,7 +468,6 @@ public void testMetadataBasedQueries()
     pqlQuery = "SELECT COUNT(*) FROM " + DEFAULT_TABLE_NAME + " WHERE DaysSinceEpoch > 16100";
     response = postQuery(pqlQuery);
     assertEquals(response.get("numEntriesScannedPostFilter").asLong(), 0);
-    assertTrue(response.get("numEntriesScannedInFilter").asLong() > 0);
```

Comment:
Same here

Reply:
My bad -- should have actually done that instead of removing. Good catch